src/styles/blocks/stop-watch.scss

```diff
@@ -0,0 +1,66 @@
+.stop-watch {
+  display: flex;
+  justify-content: center;
+  height: 400px;
+  width: 400px;
+  border: 1px solid black;
+  border-radius: 50%;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  margin-right: -50%;
+  transform: translate(-50%, -50%);
+
+  &__minute-hand {
+    position: absolute;
+    top: 50px;
+    width: 10px;
+    height: 150px;
+    background-color: blue;
+    border-radius: 5px;
+    transition-duration: 1s;
```

Comment:
Add transition-property too, because now it applies to all properties, which is not good for performance 
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x